examples/shapes/sphereCotangentLaplaceOperator.cpp

```diff
@@ -0,0 +1,336 @@
+/**
+  * This program is free software : you can redistribuate it and/or modify
+  * it under the terms of the GNU Lesser General Public License as
+  * published by the Free Software Foundation, either version 3 of the
+  * License, or (at you option) any later version.
+  *
+  * This program is distributed int the hope that it will be useful,
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  * GNU General Public License for more details.
+  *
+  * You should have received a copy of the GNU General Public License
+  * along with this program. If not, set <http://www.gnu.org/license/>.
+**/
+
+/**
+ * @file shapes/exampleTriangulatedSurface.cpp
+ * @ingroup Examples
+ * @author Thomas Caissard (\c thomas.caissard@liris.cnrs.fr )
+ * Laboratoire d'InfoRmatique en Image et Systèmes d'information - LIRIS (CNRS, UMR 5205), INSA-Lyon, France
+ * LAboratoire de MAthématiques - LAMA (CNRS, UMR 5127), Université de Savoie, France
+ *
+ * @date 2017/06/20
+ *
+ * An example file named exampleTriangulatedSurface.
+ *
+ * This file is part of the DGtal library.
+**/
+
+//! [TriangulatedSurfaceUsageFull]
+
+#define WITH_3D
```

Comment:
I put it at the correct places. It is not really useful since the CMakeLists requires both Eigen and QGLViewer for this example.